### PR TITLE
update accepted and confirmed for requests

### PR DESCRIPTION
**Pull request by mmxiong**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -88,6 +115,18 @@ app.post('/v1/request/:requestId/complete', (req, res) => {
...
 });
 
+app.get('/v1/user/:userId/requests', (req, res) => {
+    const { userId } = req.params;
+
+    const query = `SELECT * FROM Request `
+                + `WHERE requesterId="${userId}" or providerId="${userId}"`;
+
+    db.query(query, (error, results) => {
+        console.log(error || "Success");
```
Comment (bliu23) [inline server/server.js:197]: lol kinda random but just checking, this outputs same as the previous if(error)... else ... right?

File: server/server.js [hunk 2]
```diff
@@ -62,20 +63,46 @@ app.post('/v1/request/:request_id/delete', (req, res) => {
...
+        } else {
+            console.log("SUCCESS!");
+            console.log(results);
+        }
     });
 });
 
 // Example call:
-// http://localhost:3000/v1/request/create?userId="test"&title="testtitle"&description="testdescription"&location="testlocation"&timeStart="%2017-03-03%2011:11:11"&timeEnd="2017-04-04%2011:11:011"
+// http://localhost:3000/v1/request/1/accept?userId="test"&time="2017-04-04%2011:11:011"
```
Comment (bliu23) [inline server/server.js]: example should be using confirm not accept right?
http://localhost:3000/v1/request/1/**accept**?userId="test"&time="2017-04-04%2011:11:011"

**Review by bliu23 — DISMISSED** [review]

File: server/server.js (hunk 2 above)
Comment (mmxiong) [reply server/server.js]: Changed.

File: server/server.js (hunk 1 above)
Comment (mmxiong) [reply server/server.js:197]: Yes

**Review by parkandrew — APPROVED** [review]

**Review by parkandrew — APPROVED** [review]

**Pull request merged**